data/json/items/tool_armor.json

```diff
@@ -2633,5 +2633,51 @@
     "revert_to": "foodperson_mask",
     "use_action": "FOODPERSON",
     "flags": [ "OUTER", "SUN_GLASSES", "NO_TAKEOFF", "LIGHT_10", "WATCH", "NO_UNLOAD" ]
+  },
+  {
+    "id": "attached_ear_plugs_on",
+    "type": "ARMOR",
+    "name": "pair of attached ear plugs (in)",
+    "name_plural": "pairs of attached ear plugs",
+    "description": "A pair of Industrial grade ear plugs, they're attached together by some string. they're inside your ears, use them to take them out.",
+    "weight": 22,
+    "volume": 0,
+    "price": 10,
+    "material": [ "cotton" ],
```

Comment:
Are `Industrial grade ear plugs` made of cotton and not some sort of pastic?

Reply:
Ah the `ear_plug` item is made of cotton, so I suppose it's fine then.

Reply:
Most cheap Industrial grade ear plugs are made out of foam and the more expensive are made up wax or silicone, but the in-game ear plugs are made out of cotton and claim to be Industrial grade so added it to the description.

Reply:
Except the ear plugs are crafted with a chunk of plastic, not cotton.